pyrex.py

```diff
@@ -177,6 +177,10 @@ def build(args):
                 print("one of the following is relevant:")
                 print("  https://docs.docker.com/install/linux/docker-ce/ubuntu/")
                 print("  https://docs.docker.com/install/linux/docker-ce/fedora/")
+                print("")
+                print("If you don't want to install docker, please disable it by setting:")
+                print(" PYREX_DOCKER=0")
```

Comment:
This is a little bit ambiguous. The $PYREX_DOCKER environment variable needs to be exported into the pyrex process for it to be strong enough to affect both the 'pyrex build' and 'pyrex run' phases.

I've done a more explicitly worded version of this in https://github.com/garmin/pyrex/pull/10, as well as restore the former language that warns about the non-Docker path being unsupported.